src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,37 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navigation = () => {
+  const handlerActiveLink = (isActive: boolean) => {
+    return cn('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+  };
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => handlerActiveLink(isActive)}
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to="/people"
+            className={({ isActive }) => handlerActiveLink(isActive)}
+          >
```

Comment:
Here you can move the NavLink into a separate PageNavLink component (as in the example in the morning lesson). This will make the code more readable and easier to reuse.